circle.yml

```diff
@@ -29,13 +29,11 @@ test:
         ./scripts/set-dev-version.js
       fi;
     - yarn lint
+    # Limit maxWorkers to 3 to avoid OOM on CircleCI
     - yarn test-ci -- -- --maxWorkers 3
     - yarn check-lockfile
     - yarn run build-dist
     - yarn run build-deb
-    # Test that the standalone .js build works as expected
-    - ./artifacts/yarn-`./dist/bin/yarn --version`.js --version
```

Comment:
Shouldn't we still include this? We should verify that the output is what we expect, though.